### ISSUE TITLE
Issue with standard include files

### ISSUE THREAD

**Issue by Danp2** [genius257/vscode-autoit#27]
```autoit
#include <array.au3> ; this isn't working
#include "array.au3" ; this works fine, linking to "C:\Program Files (x86)\AutoIt3\Include\array.au3"
```

Settings.json file contains

```json
		"autoit3.userDefinedLibraries": [
          "C:\\Users\\danpo\\Dropbox\\Autoit UDFs"
		],
		"autoit3.installDir": "C:\\Program Files (x86)\\AutoIt3\\"
```

Observations when using `<>` as delimiters --

- No hovers, intellisense, etc
- No file link for UDF

P.S. I wanted to investigate this myself, but all my breakpoints appeared as unbound. Can you provide guidance on this?

**Comment by genius257** [comment]
https://github.com/genius257/vscode-autoit/blob/8116ca85d9f21ccbd255d174d193ffde335db4d2/server/src/autoit/Workspace.ts#L135

Seems to be caused by this line returning `null` instead of `x`, if a match was found previously.
I will try to fix and verify later today :)

>  P.S. I wanted to investigate this myself, but all my breakpoints appeared as unbound. Can you provide guidance on this?

Are you running the code using the `"Client + Server"` option from the drop-down list in the debug tab?

**Comment by Danp2** [comment]
@genius257 I tried "Client + Server", but something still seems amiss. I'm going to move this discussion to a separate issue.

**Comment by genius257** [comment]
Should be fixed in version 1.2.3 :smile:
